websites/E/Emberspark Games/presence.ts

```diff
@@ -0,0 +1,178 @@
+const presence = new Presence({ clientId: "1196469492310618142" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Icon = "https://i.imgur.com/w5E2gIj.png",
+	DesolationIcon = "https://i.imgur.com/xLRd3Aa.png",
+	ConsumerCatalog = "https://i.imgur.com/FxjlP2Z.png",
+}
+
+presence.on("UpdateData", async () => {
+	const currentPath = document.location.pathname.toLowerCase().split("/"),
+		presenceData: PresenceData = {
+			smallImageKey: Assets.Viewing,
+			smallImageText: "Viewing...",
+			startTimestamp: browsingTimestamp,
+		};
+
+	if (!currentPath[1]) {
+		presenceData.details = "Emberspark Games";
+		presenceData.largeImageKey = Assets.Icon;
+		presenceData.state = "Viewing the home page";
+		presenceData.buttons = [
+			{
+				label: "Check out our games/mods!",
+				url: "https://embersparkgames.com",
+			},
+		];
```

Comment:
[Buttons](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) that lead to main pages are not allowed.